src/Orchard/Mvc/AntiForgery/AntiForgeryAuthorizationFilter.cs

```diff
@@ -53,91 +75,35 @@ public class AntiForgeryAuthorizationFilter : FilterProvider, IAuthorizationFilt
                        .AntiForgery.Equals("enabled", StringComparison.OrdinalIgnoreCase));
         }
 
-        private static string GetArea(RouteData routeData) {
+        static string GetArea(RouteData routeData) {

```

Comment:
Yeah I think it is good practice to explicitly define scopes, and like you said, that seems to be the norm, at least in this solution.

Thank you, I'm looking forward to getting this PR merged in.